tock/projects/tests.py

```diff
@@ -494,7 +494,7 @@ def test_project_timeline(self):
         self.assertEqual(
             res['groups'],
             {
-                self.user: {
+                self.user.user_data.display_name: {
```

Comment:
in this case it may be better to use self.user.get_full_name() rather than traverse the DB to user_data